lib/Net/Async/Blockchain/ETH.pm

```diff
@@ -90,29 +89,48 @@ returns a Future, the on_done response will be the accounts array.
 
 =cut
 
-async sub accounts {
+sub accounts {
```

Comment:
is the return in the comment still accurate?

Reply:
No, it's not, I will fix the comment, thanks!